.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -12,6 +12,7 @@
 - [ ] **New feature** (non-breaking change that adds functionality)
 - [ ] **Documentation update**
 - [ ] **Breaking change** (could cause existing functionality to not work as expected)
+- [ ] **Small changes** (small non-breaking changes that does not fit any of the above. E.g changing a file or a function name)
```

Comment:
i agree that the types doesn't seem to cover all cases.

However, i think "small change" is vague and subjective. The word "Small" has different meaning to person A than person B. How do you define "small"? (I have thought about this while updating the PR template last time, but i can't think a good term...)

The example you listed `changing a file or a function name` doesn't necessarily sound small to me, change a function name could be a breaking change.

Reply:
Good point, but I also explained the changes have to be non-breaking changes. I agree the changing file/function could be a breaking change but some can be fit into this category. Maybe we should call it `Other changes` instead of `small changes`. 🤔

Reply:
@alexvuong how about "Refactor" instead? That's essentially what Kevin was doing in his PR.

For an example of what other people consider as Types Of Changes, take a look at this image here: https://github.com/commitizen/cz-cli#using-the-command-line-tool

Reply:
"other changes" sounds better than "small changes".

Reply:
"Refactor" feels too specific to me, an example is this [PR](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/83) it only adds tests.

Reply:
@vmarta `refactor` would be better if we have more specific type changes (like the one from your example). In our case, `other changes` might also include `adding test cases` or `adding some trivial changes` or just `change the building pipeline`.